src/plugins/discover/public/components/discover_tour/discover_tour_provider.tsx

```diff
@@ -0,0 +1,306 @@
+    title: i18n.translate('discover.dscTour.stepSort.title', {
+      defaultMessage: 'Sort on one or more fields',
+    }),
+    content: (
+      <FormattedMessage
+        id="discover.dscTour.stepSort.description"
+        defaultMessage="Use the column heading to sort on a single field, or the popover for multiple fields."
+      />
+    ),
+    imageName: 'sort.gif',
+    imageAltText: i18n.translate('discover.dscTour.stepSort.imageAltText', {
+      defaultMessage:
+        'Click a column header and select the desired sort order. Adjust a multi-field sort using the fields sorted pop-up.',
+    }),
+  },
+  {
+    anchor: DISCOVER_TOUR_STEP_ANCHORS.changeRowHeight,
+    title: i18n.translate('discover.dscTour.stepChangeRowHeight.title', {
+      defaultMessage: 'Change the row height',
+    }),
+    content: (
+      <FormattedMessage
+        id="discover.dscTour.stepChangeRowHeight.description"
+        defaultMessage="Adjust the number of lines to fit the contents."
+      />
+    ),
+    imageName: 'rows_per_line.gif',
+    imageAltText: i18n.translate('discover.dscTour.stepChangeRowHeight.imageAltText', {
+      defaultMessage:
+        'Click the display options icon to adjust the row height to fit the contents.',
+    }),
+  },
+  {
+    anchor: DISCOVER_TOUR_STEP_ANCHORS.expandDocument,
+    title: i18n.translate('discover.dscTour.stepExpand.title', {
+      defaultMessage: 'Expand documents',
+    }),
+    content: (
+      <FormattedMessage
+        id="discover.dscTour.stepExpand.description"
+        defaultMessage="Click {expandIcon} to view, compare, and filter documents."
+        values={{
+          expandIcon: (
+            <EuiIcon
+              size="s"
+              type="expand"
+              color="text"
+              aria-label={i18n.translate('discover.dscTour.stepExpand.expandIconAriaLabel', {
+                defaultMessage: 'Expand icon',
+              })}
+            />
+          ),
+        }}
+      />
+    ),
+    imageName: 'expand.gif',
+    imageAltText: i18n.translate('discover.dscTour.stepExpand.imageAltText', {
+      defaultMessage:
+        'Click the expand icon to inspect and filter the fields in the document and view the document in context.',
+    }),
+  },
+];
+
+const FIRST_STEP = 1;
+
+const prepareTourSteps = (
+  stepDefinitions: TourStepDefinition[],
+  getAssetPath: (imageName: string) => string
+): EuiTourStepProps[] =>
+  stepDefinitions.map((stepDefinition, index) => ({
+    step: index + 1,
+    anchor: stepDefinition.anchor,
+    title: stepDefinition.title,
+    maxWidth: MAX_WIDTH,
+    content: (
+      <>
+        <EuiText>
+          <p>{stepDefinition.content}</p>
+        </EuiText>
+        {stepDefinition.imageName && (
+          <>
+            <EuiSpacer size="m" />
+            <EuiImage
+              alt={stepDefinition.imageAltText}
+              src={getAssetPath(stepDefinition.imageName)}
+              size={300}
```

Comment:
```suggestion
size="fullWidth"
```
If you control the size of each tour step using `width` rather than `maxWidth` then set the image size to `fullWidth`, the margin on the left and right of the image will always be equal between steps - personally, this feels a bit more polished rather than having a larger margin on the right-hand side of the image than on the left-hand side 😄